go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/apparentlymart/go-cidr v1.0.0
 	github.com/google/go-cmp v0.5.9
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
-	github.com/openshift/api v0.0.0-20230120195050-6ba31fa438f2
+	github.com/openshift/api v0.0.0-20230620212129-894c5465a5ac
```

Comment:
Do you want https://github.com/openshift/api/commit/894c5465a5ac30cf99b2f226a29aebd29dab5fef or https://github.com/openshift/api/commit/b5a47f95ba705b51a3439d6a080d24117a5df468 (the merge commit from the release-4.13 branch)?  For the original PR, you used https://github.com/openshift/api/commit/333bc194ef7a13338a4343c25f81ff856ab0d0fd (the merge commit from the main branch).  It might not matter as the history is linear, but it seems like it would be good to be consistent.  

The bump https://github.com/openshift/api/compare/6ba31fa438f2...894c5465a5ac (or https://github.com/openshift/api/compare/6ba31fa438f2...b5a47f95ba70) pulls in a lot of unrelated commits, but none that looks like it would affect the DNS operator or API.

Reply:
Yea good catch. I usually do merge commits, I think I got confused because when I did the `git cherry-pick` for this PR I had to select the non-merge commit https://github.com/openshift/cluster-dns-operator/commit/a9eff7f40744019d2ccea7633ae96e75915e6f14 for cluster-dns-operator, but that's somewhat irrelevant, since we are talking about the API vendoring commit here.

Updated to use the merge commit.